common/src/main/java/dev/latvian/kubejs/item/ItemModificationProperties.java

```diff
@@ -50,4 +51,16 @@ public void setTier(Consumer<ModifiedToolTier> c) {
 			throw new IllegalArgumentException("Item is not a tool/tiered item!");
 		}
 	}
+
+	public void setFoodProperties(Consumer<FoodBuilder> consumer) {
+		Item originalItem = (Item)item;
+		FoodProperties fp = originalItem.getFoodProperties();
+		if(fp == null) {
+			throw new RuntimeException(String.format("Food %s is not edible food.", originalItem));
```

Comment:
Q: Should users be able to make non-edible things edible using Kube?

Reply:
If somebody wants to be able to eat Glass Shards from Quark, then as a user I don't see a reason to stop them, personally.

This does not mean I want to be able to eat glass, I swear. I promise I'm sane. aajajajskkshsda.